Makefile

```diff
@@ -30,7 +30,12 @@ vendor:
 	@echo "✓ Filling vendor folder with library code ..."
 	@go mod vendor
 
+INTEGRATION = gotestsum --format github-actions --rerun-fails --jsonfile output.json --packages "./integration/..." -- -parallel 4 -timeout=2h
```

Comment:
> -parallel 4

why did we set this, isn't it by default set to number of CPUs?

Reply:
It is, but we have 2 CPUs on the runners, so increased it a bit.